src/feed/Feed.js

```diff
@@ -98,6 +99,19 @@ class Feed {
                 });
                 callback({ quotes: [] });
                 return;
+            } if (errorCode === 'NoRealtimeQuotes') {
```

Comment:
this could be simplified. Try to cut down the amount of duplicate code in ur PRs